state/rds-postgres.yaml

```diff
@@ -105,7 +106,9 @@ Conditions:
   HasSSHBastionSecurityGroup: !Not [!Equals [!Ref ParentSSHBastionStack, '']]
   HasAlertTopic: !Not [!Equals [!Ref ParentAlertStack, '']]
   HasDBSnapshotIdentifier: !Not [!Equals [!Ref DBSnapshotIdentifier, '']]
-  HasEncryption: !Equals [!Ref Encryption, true]
+  CreateKey: !And
```

Comment:
Something that I discovered lately, instead of

`!Equals [!Ref Encryption, true]`

you could also reuse the existing condition

`!Condition HasEncryption`

It' snot yet used all over the templates, but I found it interesting :)

Reply:
Yeah nice, I expected this to be a thing but somehow didn't find the docs. Sorry for recent inaction, going to look at this tonight.